src/types/register_input.rs

```diff
@@ -53,6 +60,19 @@ pub enum InputRtpAudioOptions {
         /// For more information, check out [RFC](https://datatracker.ietf.org/doc/html/rfc6716#section-2.1.7).
         forward_error_correction: Option<bool>,
     },
+
+    Aac {
+        /// The config should be encoded as described in [RFC 3640](https://datatracker.ietf.org/doc/html/rfc3640#section-4.1)
```

Comment:
- Missing `.` at the end
- This is rendered as one big blog of text, Maybe add a newline after it.
- I don't think this explanation is good enough
  - You are recommending passing an additional flag to ffmpeg command, but additional to what?
  - What if someone is using GStreamer to deliver RTP, even if you don't know how to access that value with gstreamer explain it. For example, instead of explaining how to get it with gstreamer you could explain how to get it from hls, rtmp, mp4/mp3 or some other container file